src/widgetastic_patternfly/__init__.py

```diff
@@ -454,16 +455,19 @@ def select(self, text=None, **kwargs):
         """Select/click an item from the menu"""
         if text:
             # Select an item based on the text of that item
-            xpath = self.TEXT_MATCHING.format(txt=quote(text))
+            try:
+                link = self.browser.element(self.TEXT_MATCHING.format(txt=quote(text)))
+            except NoSuchElementException:
+                link = self.browser.element(self.PARTIAL_TEXT.format(txt=quote(text)))
```

Comment:
I believe we should make the choice explicit.

Reply:
done